src/main/kotlin/net/ccbluex/liquidbounce/utils/item/ItemExtensions.kt

```diff
@@ -159,15 +160,18 @@ val ItemStack.attackDamage: Double
          * see https://bugs.mojang.com/browse/MC-196250
          *
          * We now use the following formula to calculate the damage:
-         * https://minecraft.wiki/w/Sharpness -> 0.5 * level + 0.5.
+         * https://minecraft.wiki/w/Sharpness
+         * >= 1.9 -> 0.5 * level + 0.5
+         * else -> 1.25 * level
          */
         return entityBaseDamage + baseDamage + getSharpnessDamage()
     }
 
 val ItemStack.sharpnessLevel: Int
     get() = EnchantmentHelper.getLevel(Enchantments.SHARPNESS.toRegistryEntry(), this)
 
-fun ItemStack.getSharpnessDamage(level: Int = sharpnessLevel) = if (level == 0) 0.0 else 0.5 * level + 0.5
+fun ItemStack.getSharpnessDamage(level: Int = sharpnessLevel) =
+    if (isOlderThanOrEqual1_9) (if (level == 0) 0.0 else 0.5 * level + 0.5) else level * 1.25
```

Comment:
One line functions are for one liners. Here you fit at least 4 lines into one line which isn't ideal.

```suggestion
fun ItemStack.getSharpnessDamage(level: Int = sharpnessLevel): Double {
    if (isOlderThanOrEqual1_9) {
        return when (level) {
            0 -> 0.0
            else -> 0.5 * level + 0.5
        }
    } else {
        return level * 1.25
    }
}
```